src/cata_tiles.cpp

```diff
@@ -2485,9 +2491,11 @@ bool cata_tiles::draw_sprite_at(
     int height = 0;
     std::tie( width, height ) = sprite_tex->dimension();
 
+    const point *offset = tile_retracted ? &tile.offset_retracted : &tile.offset;
```

Comment:
You don't ned to use a pointer here, a reference will do.
```suggestion
    const point &offset = tile_retracted ? tile.offset_retracted : tile.offset;
```

Reply:
Thanks for the hint!